frontend/src/components/containers/ContainersListContent.js

```diff
@@ -51,7 +52,7 @@ const getTableColumns = (samplesByID, containersByID, containerKinds) => [
             {samples.map((id, i) =>
               <React.Fragment key={id}>
                 <Link to={`/samples/${id}`}>
-                  {withSample(samplesByID, id, sample => sample.name, <span>Loading…</span>)}
+                  {samplesByID[id]?.name ?? "Loading..."}
```

Comment:
nope. withItem() fetches the missing data ...

Reply:
Not the best place to fetch missing data.